### ISSUE TITLE
voila-vuetify faills for voila 0.1.15

### ISSUE THREAD

**Issue by JovanVeljanoski** [voila-dashboards/voila-vuetify#18]
Hi,

You are probably aware of this, but just to report that `voila-vuetify` fails for the latest version of `voila 0.1.15`

Example:
```
 voila --template vuetify-default simple_example.ipynb

Traceback (most recent call last):
  File "/Users/jovan/miniconda3/bin/voila", line 10, in <module>
    sys.exit(main())
  File "/Users/jovan/miniconda3/lib/python3.7/site-packages/traitlets/config/application.py", line 663, in launch_instance
    app.initialize(argv)
  File "/Users/jovan/miniconda3/lib/python3.7/site-packages/voila/app.py", line 348, in initialize
    self.setup_template_dirs()
  File "/Users/jovan/miniconda3/lib/python3.7/site-packages/voila/app.py", line 368, in setup_template_dirs
    assert conf['traitlet_configuration']['base_template'] == self.voila_configuration.template
KeyError: 'traitlet_configuration'
```

I hope this helps,

Cheers!

**Comment by SylvainCorlay** [comment]
cc @mkcor should we keep that `assert` statement?

**Comment by mkcor** [comment]
Well, I admit it's not fair to keep it if we don't document what the template config file should look like.  That's a way to enforce a standard for Voilà template config files, but

* the conversation to come up with it only happened [there](https://github.com/voila-dashboards/voila/pull/326#issuecomment-526417088), maybe this should be a broader conversation;
* once we agree on the standard, we should document it (at the moment, it's only mentioned in the `voila-reveal` announcement [blog post](https://blog.jupyter.org/a-slideshow-template-for-voil%C3%A0-apps-435f67d10b4f).

@JovanVeljanoski don't you have a `conf.json` file under `PREFIX/share/jupyter/voila/templates/voila-vuetify/`? Does it look like this:
```
{
  "traitlet_configuration": {
    "base_template": "vuetify-default",
    (...)
  }
}
```

**Comment by JovanVeljanoski** [comment]
@mkcor 

Yes, i do, but the file only contains this:

```
{"base_template": "vuetify-base"}
```

**Comment by mkcor** [comment]
@JovanVeljanoski Makes sense. If you don't want to wait for #19 to be merged and `voila-vuetify` to be re-released on `conda-forge`, edit this `conf.json` file into
```
{
  "traitlet_configuration": {
    "base_template": "vuetify-default"
  }
}
```
for the moment.

Thanks for reporting, by the way!

**Comment by JovanVeljanoski** [comment]
Thank you @mkcor  and @SylvainCorlay for your swift reply. Feel free to close this when you see fit.

**Comment by SylvainCorlay** [comment]
@JovanVeljanoski we have fixed the issue with voila 0.1.16

Thanks for reporting.
